interface.go

```diff
@@ -330,3 +330,41 @@ type AcceptPayableChecker interface {
 	SetPayableChecker(payableHandler PayableChecker) error
 	IsInterfaceNil() bool
 }
+
+// EnableEpochsHandler is used to verify which flags are set in the current epoch based on EnableEpochs config
+type EnableEpochsHandler interface {
```

Comment:
this can be a subset with only required getters in order to simplify the code

Reply:
it is a subset.. added new methods to it